proposals/2021/Add AuditFilingStatus.md

```diff
@@ -0,0 +1,52 @@
+# Add AuditFilingStatus
+
+## Overview
+
+This proposal
+
+## Justification
+
+This field is only specific to Atlanta reports, but could be a useful input for others if requested.
```

Comment:
hey Jie - I think just clarify with this justification section that this originally came about in working directly with the city of Atlanta, but we believe it is a good field and therefore are adding it to the schema...something like that.